frontend/src/metabase/search/components/filters/CreatedByFilter/UserListElement.tsx

```diff
@@ -0,0 +1,28 @@
+import type { UserListResult } from "metabase-types/api";
+import { UserElement } from "metabase/search/components/filters/CreatedByFilter/CreatedByContent.styled";
+import { Text } from "metabase/ui";
+
+export type UserListElementProps = {
+  value: UserListResult;
+  onClick: (value: UserListResult) => void;
+  isSelected: boolean;
+};
+
+export const UserListElement = ({
+  value,
+  isSelected,
+  onClick,
+}: UserListElementProps) => (
+  <UserElement
+    data-testid="user-list-element"
+    onClick={() => onClick(value)}
+    isSelected={isSelected}
+    data-is-selected={isSelected}
+    px="sm"
+    py="xs"
+  >
+    <Text weight={700} color={isSelected ? "brand.1" : undefined}>
+      {value.common_name || `${value.first_name} ${value.last_name}`}
```

Comment:
If user is called `Anna Konstantynopolitańczykiewiczówna` there will be an ugly horizontal scrollbar.
Let's ellipsify it?

![image](https://github.com/metabase/metabase/assets/6830683/bb53cf2a-5fe0-40e3-8a0d-7cfa6d1df2f6)

Reply:
Done, tested with 100 `Anna Konstantynopolitańczykiewiczówna`s